src/components/helpers/selectors.js

```diff
@@ -15,7 +15,7 @@ const isMobileAndTabletType = () => {
 };
 
 const isEdgeChromiumType = () => {
-  if (os.name === OS_TYPES.WINDOWS && os.version === '10') {
+  if (os.name === OS_TYPES.WINDOWS) {
```

Comment:
What a coincidence, I made an Edge-related PR too! #111.

Should the OS check be for all OS` since Chromium Edge runs on Windows, Mac, and Linux?

Reply:
😯 Didn't know there were mac and linux versions. I wonder how many use those. Will change.

Reply:
Yup! Also on Android and iOS (not sure how the user agents change with that). I use to work on the Edge team 😛 .